opentreemap/treemap/templates/treemap/edits.html

```diff
@@ -72,7 +72,7 @@
           </td>
           <td>
             {% if audit.model|is_filterable_audit_model %}
-            <a href="?models={{ audit.model }}&model_id={{ audit.model_id }}">
+            <a href="?models={{ audit.model }}&model_id={{ audit.model_id|unlocalize }}">
```

Comment:
This is not a problem on `master` because of the `{% localize off %}` block above. It does not look like it does any harm to add this, though.